cli/flags.go

```diff
@@ -862,6 +862,13 @@ var (
 			Usage: "name of the organization",
 		},
 	}
+	FlUserID = StringFlag{
```

Comment:
I think we should call it username, because id suggests me it's an integer value. On the login screen we also call it username.

Reply:
We already have a username flag for authentication. 
This piece of data is called userId everywhere else on the API and in the cli. (e.g. the result of  'cb user list', 'cb org list' commands)